docs/source/execute_api.rst

```diff
@@ -0,0 +1,143 @@
+Executing notebooks
+===================
+
+.. module:: nbconvert.preprocessors
+
+In this section we show how to execute a ``.ipynb`` notebook
```

Comment:
Introduction paragraph:

Jupyter notebooks are often saved with cleared output cells. `nbconvert` provides a helpful way to execute the input cells of an `.ipynb` notebook file and save the results, both input and output cells, as an `.ipynb` file. This conversion can be very helpful when teaching using notebooks or exploring notebooks created by other researchers.